recipe/meta.yaml

```diff
@@ -32,6 +32,7 @@ requirements:
 test:
    commands:
      - make_watertight --help
+     - test -f "${PREFIX}/bin/make_watertight"
```

Comment:
Shouldn't this go in the other order?  If it can't find this file, then it can't do the previous line, either